web/src/js/components/Authentication/Authentication.js

```diff
@@ -348,6 +371,39 @@ class Authentication extends React.Component {
                   <span>Your account has been deleted.</span>
                 </Typography>
               ) : null}
+              {this.state.loggedOutMessage !== LOGGED_OUT_MESSAGE_TYPE.NONE ? (
+                <div
+                  data-test-id={'logged-out-message'}
+                  style={{
+                    backgroundColor: 'rgb(255, 244, 229)',
```

Comment:
Open to different colors for this alert and for text. I copied that of the MUI alert component.